Sources/DoubleExtensions.swift

```diff
@@ -14,6 +14,9 @@ extension Double {
 
     /// EZSE: Converts Double to Int
     public var toInt: Int { return Int(self) }
+    
+    /// EZSE: Converts Double to CGFloat
+    public var toCGFloat: CGFloat { return CGFloat(self) }
```

Comment:
Ill use `greatestFiniteMagnitude` method which will guard against this.

Will this be ok for a test?